BioCrypt-Personal-V0.4 Source-code/main.py

```
@@ -0,0 +1,1814 @@
+    def get_user_info():
+        """ Load user info or prompt the user to enter it using a GUI. """
+        contract_address, account_address, private_key = load_user_info()
+        if not contract_address or not account_address or not private_key:
+            ask_user_info()  # Prompt the user to enter their info
+            contract_address, account_address, private_key = load_user_info()  # Reload after saving
+        return contract_address, account_address, private_key
+
+    def upload_to_ipfs(file_path):
+        """ Upload a file to IPFS and return its CID. """
+        with open(file_path, "rb") as file:
+            response = requests.post(
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)